bogenliga/bogenliga-application/src/main/java/de/bogenliga/application/springconfiguration/security/jsonwebtoken/JwtTokenProvider.java

```diff
@@ -180,13 +180,22 @@ private String createToken(final String username, final Long id, final Long vers
         final Date now = new Date();
         final Date validity = new Date(now.getTime() + validityInMilliseconds);
 
+        // if username is ligadefault, remove validity from jwts builder
+        if(username.equals("ligadefault")) {
+            return Jwts.builder()
+                    .setClaims(claims)
+                    .setIssuedAt(now)
+                    .signWith(SignatureAlgorithm.HS256, secretKey)
+                    .compact();
+        } else {
```

Comment:
Asks whether the use is a default user and creates its according JWT (w/o expiration)